website/docs/r/linux_web_app.html.markdown

```diff
@@ -218,13 +218,13 @@ An `auth_settings` block supports the following:
 
 An `auth_settings_v2` block supports the following:
 
-* `auth_enabled` - (Optional) Should the AuthV2 Settings be enabled. Defaults to `false`.
+* `auth_enabled` - (Optional) Should the AuthV2 Settings be enabled.
```

Comment:
May we add a `Default to 'false'` to all boolean optional fields which has not set a default value in schema?

Reply:
I have upated the tool to skip such case, for the boolean field, if the schema has not specified the default value as `false`, but the document contans `default to 'false'`, the tool will do nothing about it.

Reply:
> May we add a `Default to 'false'` to all boolean optional fields which has not set a default value in schema, Or we should give users a perception that bool type fields without explicitly stated default values are all defaulted to false?

I'm unsure about this - since sometimes if the property maps to a property in the SDK that is optional we sometimes have to omit sending it because of how the API behaves. For the time being it's fine for the tool to just skip over existing instances of this.